pages/cart.js

```diff
@@ -0,0 +1,36 @@
+const { I } = inject();
```

Comment:
in `pages` folder has 1 file ends with Page, but other files are not. make them similar